qt/RpcQueue.h

```diff
@@ -0,0 +1,90 @@
+/*
+ * This file Copyright (C) 2015 Mnemosyne LLC
+ *
+ * It may be used under the GNU GPL versions 2 or 3
+ * or any future license endorsed by Mnemosyne LLC.
+ *
+ * $Id$
+ */
+
+#ifndef QTR_RPC_QUEUE_H
```

Comment:
Include guards were replaced with `#pragma once` recently, please use it here as well.